pkg/imager/fyne.go

```diff
@@ -157,11 +158,16 @@ func (fg *FyneUI) build() {
 
 	switch fg.imgLoc {
 	case fg.locations[0]:
-
-		// Download section.
+		ctx, cancelF := context.WithCancel(context.Background())
```

Comment:
What do you think of renaming `cancelF` to `cancel` for consistency with other usages of `context.WithCancel`?